en/style/typography.md

```diff
@@ -30,42 +34,64 @@ These styles also determine the type of text element and typography class that w
   - `igx-typography__caption` for the caption typography style.
   - `igx-typography__overline` for the overline typography style.
 
-These styles can be modified further via the Sketch plugin to change the look of those elements via theming on the application level.
+These styles can be modified further via the **Indigo.Design** plugin to change the look of those elements via theming on the application level.
 
 Use these when creating intricate layouts for articles, blog posts etc. The Text Styles in the **Indigo.Design System** are meant to help you define a consistent theme and style for all the texts in your designs.
 
 > [!Warning]
-> In Sketch, there is also a collection of `Internal` text styles used within the Components for validation and special cases. These styles are not supposed to be inserted on your artboards directly but exist to assure typographic consistency for the Components within which they are used.
+> In Figma and Sketch, there is also a collection of `Internal` text styles used within the Components for validation and special cases. These styles are not supposed to be inserted on your artboards directly but exist to assure typographic consistency for the Components within which they are used.
```

Comment:
In Figma we don't have internal text styles I believe. These I believe are the ones which are used on the avatar with initials, slider etc and in Figma we didn't define these as styles, like it was done in Sketch. As for detail 1 and detail 2 I'm not sure whether they fit in this category, what do you think @sdimchevski? I guess that's why they were left out above?